src/main/java/domain/status/PaymentType.java

```diff
@@ -0,0 +1,22 @@
+package domain.status;
+
+import java.util.Arrays;
+import util.ExceptionMessage;
+
+public enum PaymentType {
+
+    CARD("1"), CASH("2");
```

Comment:
PaymentType에서 할인율을 같이 관리하면 어떨까요?
두 가지는 서로 연관된 값이고, 추후 다른 결제 방법에 따라 할인율 정책이 추가될 수도 있습니다.
```suggestion
    CARD("1", 1),
    CASH("2", 0.95);
```
위와 같은 이유로 다음과 같이 정보를 한꺼번에 관리하고,
할인율 계산하는 로직을 Payment 객체에서 수행하는 설계를 추천드리고 싶습니다:)

Reply:
@Ridealist 오.. 엄청 좋은 것 같아요 곧바로 바꿔보도록 하겠습니다 !_!_!